MAUI/TextInputLayout/States-And-Colors.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: post
+title: States and Colors in .NET MAUI Text Input Layout control | Syncfusion
+description: Learn here all about States and Colors support in Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# States and Colors in .NET MAUI Text Input Layout (SfTextInputLayout)
+
+Use the visual state manager to change the `Stroke` properties based on the visual states set from code. The applicable visual states are normal, focused, and error.
+
+Based on the states, the colors will be applied to the floated hint label and borders. So, when the input view is in focused state, the focused state `Stroke` will be applied; it is similar to other states also. The current stroke can be obtained from the `CurrentActiveColor` property.
+
+N> The error color will not be set to `CurrentActiveColor` when `HasError` property is set to `true`.
+
+## Stroke
+The `Stroke` property value will be applied to the hint label and border of SfTextInputLayout. 
```

Comment:
The `Stroke` property value will be applied to the hint label and border of the SfTextInputLayout.

Reply:
Addressed the changes